### PR TITLE
Fix up the speech audio manager

### PR DESCRIPTION
**Pull request by duncte123**
- [x] I have tested my changes.
- [ ] I didn't test my changes. I will do that in the following 7 days and agree that this Pull request will be closed if I don't test it in the named time span.
- [x] I know the language good enough for providing high-quality code.

Changes proposed in this pull request:
---

- Fixed up the speech manager
 
- Removed the dbots token from the config

### CODE REVIEW

File: src/main/java/ml/duncte123/skybot/objects/audiomanagers/speech/SpeechAudioSourceManager.java [hunk 1]
```diff
@@ -78,15 +76,9 @@ public AudioItem loadItem(DefaultAudioPlayerManager manager, AudioReference refe
...
-
-            String encoded;
+                    /*.substring(0, Math.min(data.length() - 1, limit))*/;
 
-            try {
-                encoded = URLEncoder.encode(data, "UTF-8");
-            } catch (UnsupportedEncodingException e) {
-                throw new FriendlyException("Could not encode data", FriendlyException.Severity.FAULT, e);
-            }
+            String encoded = URLEncoder.encode(data, StandardCharsets.UTF_8);
```
Comment (ramidzkh) [inline src/main/java/ml/duncte123/skybot/objects/audiomanagers/speech/SpeechAudioSourceManager.java]: Java 10 feature, wasn't showing up for me

**Comment by duncte123** [comment]
Well time to install Java 10 then 😜

**Comment by ramidzkh** [comment]
> Well time to install Java 10 then 😜

Gradle's using Java 10, Java 10 installed and set as default, I don't know why IntelliJ wasn't suggesting that to me.

**Comment by duncte123** [comment]
Oof XD

**Pull request merged**